library/src/scala/tasty/reflect/CompilerInterface.scala

```diff
@@ -148,6 +148,18 @@ trait CompilerInterface {
   def Context_GADT_addToConstraint(self: Context)(syms: List[Symbol]): Boolean
   def Context_GADT_approximation(self: Context)(sym: Symbol, fromBelow: Boolean): Type
 
+  /** Get package symbol if package is either defined in current compilation run or present on classpath. */
+  def Context_requiredPackage(self: Context)(path: String): Symbol
+
+  /** Get class symbol if class is either defined in current compilation run or present on classpath. */
+  def Context_requiredClass(self: Context)(path: String): Symbol
+
+  /** Get module symbol if module is either defined in current compilation run or present on classpath. */
+  def Context_requiredModule(self: Context)(path: String): Symbol
```

Comment:
The meta-programmer may not know what's a module. Maybe rename to `requiredObject`?

Reply:
Better to keep names consistent. It will be simpler for documentation. We also have `moduleClass` and `companionModule` that use the same naming scheme.

Reply:
I totally agree, for advanced meta-programming, keeping the concepts closer to the compiler will help both meta-programmers who want to get into the compiler and maintenance of the framework.